### ISSUE TITLE
Hyperparameters tuning for other datasets

### ISSUE THREAD

**Issue by dineshdaultani** [shiyegao/DDA#13]
Hi DDA authors,
I am currently trying to do parameter tuning of the following parameters for other datasets:
- Diffusion range - N
- Scaling factor - D
- Refinement range - W

However, it seems like there's no discussion or reasoning behind selecting default value 50 for diffusion range, could you please explain how do you select this parameter value?

Also, could you let me know if you tried tuning these above parameters, and having any thoughts on the same if using FID metrics let's say rather than corruption dataset accuracy?

**Comment by shiyegao** [comment]
In our paper, we delve into the concept of **Diffusion range - N** in Section 3.2. For a more in-depth exploration, our paper appendix includes a comprehensive ablation study on fine-tuning both the **Scaling factor - D** and **Refinement range - W**.

To access the detailed information, please refer to the following link: [Paper Link](https://arxiv.org/pdf/2207.03442.pdf).

In terms of evaluation metrics, we acknowledge that FID can indeed capture crucial characteristics of the images generated by our DDA, such as authenticity. However, it is essential to emphasize that, within our specific context, our primary focus is on effectiveness, utility, and accuracy.
